lib/ansible/modules/cloud/gandi/gandi_vps.py

```diff
@@ -272,7 +268,7 @@ def stop_instances(module, driver, instance_names):
 
             except GandiException as e:
                 msg = 'Unexpected error when starting instance %s' % name
-                msg = msg + e
+                msg = msg + str(e)
```

Comment:
Why this str(e) ?

Side note, variable name should be at least 3 characters -> rename e to exc for better python style

Reply:
If I remember correctly I had trouble concatenating without an explicit cast to string.